controllers/replication.storage/volumereplication_controller.go

```diff
@@ -614,6 +633,34 @@ func (r *VolumeReplicationReconciler) enableReplication(vr *volumeReplicationIns
 	return nil
 }
 
+// getVolumeReplicationInfo gets volume replication info.
+func (r *VolumeReplicationReconciler) getVolumeReplicationInfo(vr *volumeReplicationInstance) (int64, error) {
+	volumeReplication := replication.Replication{
+		Params: vr.commonRequestParameters,
+	}
+
+	resp := volumeReplication.GetInfo()
+	if resp.Error != nil {
+		vr.logger.Error(resp.Error, "failed to get volume replication info")
+
+		return 0, resp.Error
+	}
+
+	infoResponse, ok := resp.Response.(*proto.GetVolumeReplicationInfoResponse)
+	if !ok {
+		err := fmt.Errorf("received response of unexpected type")
+		vr.logger.Error(err, "unable to parse response")
+
+		return 0, err
+	}
+
+	var last_sync_time int64
+
+	last_sync_time = infoResponse.LastSyncTime
+
+	return last_sync_time, nil
+}
+
```

Comment:
Just returned .LastSyncTime no need to have a variable

Reply:
the code has been cleaned up and this is addressed now